docs/one-click-integrations/pagerduty-integration.md

```diff
@@ -16,3 +18,34 @@ To set up the PagerDuty integration:
 1.  Follow the instructions on the Pulse UI to create a new webhook on PagerDuty using the provided URL. For example:
 
     ![PagerDuty webhook](images/pagerduty-webhook.png)
+
+## Collected data
+
+The table below lists the data that the PagerDuty integration collects from your PagerDuty webhooks, together with:
+
+-   The mapping between the data collected from PagerDuty and the [Pulse data model](https://ingestion.pulse.codacy.com/v1/api-docs#tocs_event)
+-   The metrics that Pulse calculates from the data to display on the dashboards
+
+<table>
+<thead>
+<tr>
+<th><strong>Data collected from PagerDuty</strong></th>
+<th><strong>Mapping to Pulse data model</strong></th>
+<th><strong>Used in</strong></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+    <td>Incident</td>
```

Comment:
@rtfpessoa is this what it is called in PagerDuty?

Reply:
Yes, this is the correct concept name on PagerDuty:

https://response.pagerduty.com/getting_started/#define-what-an-incident-and-major-incident-are-for-you

https://response.pagerduty.com/before/what_is_an_incident/